ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -585,12 +585,21 @@ open class NavigationBar: UINavigationBar {
         let button = BadgeLabelButton(type: .system)
         button.item = item
         button.shouldUseWindowColorInBadge = style != .system
+
+        let insets: NSDirectionalEdgeInsets
         if isLeftItem {
```

Comment:
isleftitem should be tested with "X" when in selection mode showing up on the left side of the navigation?

Reply:
Just added screenshots of the selection mode in the description.